app/src/main/java/fr/neamar/kiss/UiTweaks.java

```diff
@@ -49,6 +52,40 @@ static void updateThemePrimaryColor(Activity activity) {
         }
     }
 
+    static void applyRoundedCorners(MainActivity mainActivity) {
```

Comment:
when you rebase, you may want to put  this code in the Interface forwarder (you get access to the mainActivity and SharedPreferences for free)

Reply:
I already have a branch that has this rebased, that's what I use on my phone ATM. I just need to double check.